packages/vite/src/node/server/index.ts

```diff
@@ -828,6 +825,9 @@ export async function _createServer(
       if (isDepsOptimizerEnabled(config, false)) {
         await initDepsOptimizer(config, server)
       }
+      if (isDepsOptimizerEnabled(config, true)) {
```

Comment:
Maybe we can do this in parallel with the web one? Or we can also move this to `transformRequest` if `ssr` is `true`

Reply:
Ya, probably better to push it to `transformRequest` 👍🏼